src/component/splits/column.rs

```diff
@@ -145,6 +146,7 @@ pub fn update_state(
     state: &mut ColumnState,
     column: &ColumnSettings,
     timer: &Snapshot<'_>,
+    splits: &SplitsSettings,
```

Comment:
I'm not particularly liking that this name doesn't imply that these are the settings.

Reply:
Yeah, I wasn't sure what to name it either, but ended up mimicking the `column: &ColumnSettings`. Is `splits_settings` better?

Reply:
Oh, yeah maybe rename both then.